ui/togeather/src/app/_models/lobby.ts

```diff
@@ -0,0 +1,10 @@
+import { Coordinates } from '../map/map.component';
+import { Restaurant } from './restaurant';
+
+export class Lobby {
```

Comment:
I believe you meant this to be an `interface` not a `class`? Same with `Restaurant` class